File: 53/solution_v3.cpp
class Solution {
public:
    int maxSubArray(vector<int>& nums) {
        int max_sum = INT_MIN, cur_sum = 0;
        for(int val : nums)
        {
            cur_sum += val;
            max_sum = max(max_sum, cur_sum);
            if(cur_sum < 0)
            {
                cur_sum = 0;
            }
        }
        return max_sum;

    }
};

File: 69/solution_v3.py
class Solution:
    def mySqrt(self, x: int) -> int:
        if x<2:
            return x

        a = 2
        while (a*a) < x :
            a+=1
        if (a*a) > x:
            return a-1
        elif (a*a) == x:
            return a